1.10/Dockerfile

```diff
@@ -20,14 +25,16 @@ Nginx server image can be extended using source-to-image tool."
 LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="Nginx 1.10" \
+      io.k8s.display-name="Nginx $NGINX_VERSION" \
       io.openshift.expose-services="8080:http" \
       io.openshift.expose-services="8443:https" \
-      io.openshift.tags="builder,nginx,rh-nginx110" \
-      com.redhat.component="rh-nginx110-docker" \
-      name="centos/nginx-110-centos7" \
-      version="1.10" \
-      release="1"
+      io.openshift.tags="builder,$NAME,rh-$NAME$NGINX_SHORT_VER" \
+      com.redhat.component="rh-$NAME$NGINX_SHORT_VER-docker" \
+      name="centos/$NAME-$NGINX_SHORT_VER-centos7" \
+      version="$NGINX_VERSION" \
+      release="$RELEASE" \
+      architecture="$ARCH" \
```

Comment:
We don't actually need to define architecture for RHEL and CentOS images, please, remove it, since it would only make us troubles once containers work on other arches.